libraries/provision/ansible/playbooks/install-couchbase-server-package.yml

```diff
@@ -69,7 +69,7 @@
       register: cb_major_version
 
     - name: COUCHBASE SERVER | Wait for node to be listening on port 8091
-      wait_for: port=8091 delay=5 timeout=30
+      wait_for: port=8091 delay=5 timeout=120
```

Comment:
isn't a bug if server does not come up in 30 seconds? 2 minutes will be more , right?

Reply:
It is intermittent, it could be overloaded network/hypervisor. It is possible that it started up before 30s but we timed out trying for the open port due to load. We've had similar issues with SG. We went from 120 to 30 for SG and saw lot of timeout issues. Bumping it back to 120 made all timeouts disappear.